src/Sylius/Bundle/ShopBundle/EventListener/UserRegistrationListener.php

```diff
@@ -82,4 +117,18 @@ private function handleUserVerificationToken(ShopUserInterface $user)
 
         $this->eventDispatcher->dispatch(UserEvents::REQUEST_VERIFICATION_TOKEN, new GenericEvent($user));
     }
+
+    /**
+     * @param ShopUserInterface $user
+     */
+    private function verifyAndLogin(ShopUserInterface $user)
+    {
+        $user->setVerifiedAt(new \DateTime());
```

Comment:
I understood this task slightly differently, that this feature enables account, user is logged after registration and there is no need to send verification email... But of course, I will change that :)